app/assets/controllers/mass_action_form_controller.js

```diff
@@ -0,0 +1,19 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+
+    submit () {
+        const confirmMessage = this.element.dataset.messageConfirm;
+        const formData = new FormData(this.element);
+        const checkboxesTargetClass = this.element.dataset.checkboxTargetClass;
+        const checkboxes = document.querySelectorAll(`input[type='checkbox'].${checkboxesTargetClass}:checked`);
```

Comment:
J’essaye d’éviter un maximum d’utiliser les classes CSS pour manipuler des éléments avec du JS, et je préfère passer par des attributs `data-` (ça offre une "API" globalement plus stable). Aussi, je ferais en sorte de privilégier des noms d’attributs qui font références à `mass-action`). Ici je ferais un truc du genre :


```suggestion
        const checkboxesGroup = this.element.dataset.massActionGroup;
        const checkboxes = document.querySelectorAll(`input[type='checkbox'][data-mass-action-group="${checkboxesGroup}":checked`);
```